pool/src/auctions/user_liquidation_auction.rs

```diff
@@ -58,12 +57,12 @@ pub fn create_user_liq_auction_data(
 
     let est_withdrawn_collateral = position_data
         .liability_raw
-        .fixed_mul_floor(percent_liquidated_i128, oracle_scalar)
+        .fixed_mul_floor(percent_liquidated_i128, SCALAR_7)
         .unwrap_optimized()
         .fixed_mul_floor(est_incentive, SCALAR_7)
         .unwrap_optimized();
     let mut est_withdrawn_collateral_pct = est_withdrawn_collateral
-        .fixed_div_ceil(position_data.collateral_raw, oracle_scalar)
+        .fixed_div_ceil(position_data.collateral_raw, SCALAR_7)
         .unwrap_optimized();
```

Comment:
were test cases added for these bugs?

Reply:
Note - a comment might be useful here too for `est_withdrawn_collateral_pct` as it's not obvious the SCALAR_7 actually reverts the result back to being 7 decimals scaled.

TBH - it might be clearer to just not do this and check the collateral_pct against the oracle scalar

Reply:
`test_create_auction_weird_scalar` also tests these changes

Agreed on collateral_pct against oracle scalar, updating